.travis.yml

```diff
@@ -1,8 +1,25 @@
 language: python
-python:
-  - "2.7"
-  - "3.4"
-  - 3.5
 install: pip install -e .
 script: py.test
-sudo: false
+
+python:
+    - "pypy"
+    - "pypy3"
+    - "2.7"
+    - "3.3"
+    - "3.4"
+    - "3.5"
+    - "3.6"
+
+matrix:
+  include:
+    - python: "3.7"
+      dist: xenial
+      sudo: true
+      install: pip install -e .
+      script: py.test
```

Comment:
Also, I don't think these sections need the install & script lines - Travis should use the top-level ones for all matrix items that don't override them. Sorry for not noticing that before.

Reply:
No worries, I'll check, ...